File: API/Event(API)/Event(API)/DTOs/NewHolderDTO.cs
using System.ComponentModel.DataAnnotations;

namespace Event_API_.DTOs
{
    public class NewHolderDTO
    {
        [Required]
        [StringLength(120)]
        public string Name { get; set; }

        public DateTime StartingDate { get; set; }
        public string Introduction { get; set; }
        public IFormFile Picture { get; set; }
    }
}

File: Client/src/app/_model/newHolderDTO.ts
export interface newHolderDTO {
  name: string;
  startingDate: Date;
  picture: File;
  Introduction: string;
}

export interface holderDTO {
  id: number;
  name: string;
  startingDate: Date;
  picture: string;
  Introduction: string;
}

export interface holderEventDto{
  id: number;
  name: string;
  contactperson:string;
  picture:string;
}
